mobilitydb/src/general/temporal_compops.c

```diff
@@ -47,7 +49,594 @@
+Datum
+Ever_eq_temporal_base(PG_FUNCTION_ARGS)
+{
+  return EAcomp_temporal_base(fcinfo, &ever_eq_temporal_base);
+}
+
+PGDLLEXPORT Datum Always_eq_base_temporal(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Always_eq_base_temporal);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a base value is always equal to a temporal value
+ * @sqlfn always_eq()
+ * @sqlop @p %=
+ */
+Datum
+Always_eq_base_temporal(PG_FUNCTION_ARGS)
+{
+  return EAcomp_base_temporal(fcinfo, &always_eq_temporal_base);
+}
+
+PGDLLEXPORT Datum Always_eq_temporal_base(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Always_eq_temporal_base);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a temporal value is always equal to a base value
+ * @sqlfn always_eq()
+ * @sqlop @p %=
+ */
+Datum
+Always_eq_temporal_base(PG_FUNCTION_ARGS)
+{
+  return EAcomp_temporal_base(fcinfo, &always_eq_temporal_base);
+}
+
+PGDLLEXPORT Datum Ever_ne_base_temporal(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Ever_ne_base_temporal);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a base value is ever different from a temporal value
+ * @sqlfn ever_ne()
+ * @sqlop @p ?<>
+ */
+Datum
+Ever_ne_base_temporal(PG_FUNCTION_ARGS)
+{
+  return EAcomp_base_temporal(fcinfo, &ever_ne_temporal_base);
+}
+
+PGDLLEXPORT Datum Ever_ne_temporal_base(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Ever_ne_temporal_base);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a temporal value is ever different from a base value
+ * @sqlfn ever_ne()
+ * @sqlop @p ?<>
+ */
+Datum
+Ever_ne_temporal_base(PG_FUNCTION_ARGS)
+{
+  return EAcomp_temporal_base(fcinfo, &ever_ne_temporal_base);
+}
+
+PGDLLEXPORT Datum Always_ne_base_temporal(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Always_ne_base_temporal);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a base value is always different from a temporal value
+ * @sqlfn always_ne()
+ * @sqlop @p %<>
+ */
+Datum
+Always_ne_base_temporal(PG_FUNCTION_ARGS)
+{
+  return EAcomp_base_temporal(fcinfo, &always_ne_temporal_base);
+}
+
+PGDLLEXPORT Datum Always_ne_temporal_base(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Always_ne_temporal_base);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a temporal value is always different from a base value
+ * @sqlfn always_ne()
+ * @sqlop @p %<>
+ */
+Datum
+Always_ne_temporal_base(PG_FUNCTION_ARGS)
+{
+  return EAcomp_temporal_base(fcinfo, &always_ne_temporal_base);
+}
+
+/*****************************************************************************/
+
+PGDLLEXPORT Datum Ever_lt_base_temporal(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(Ever_lt_base_temporal);
+/**
+ * @ingroup mobilitydb_temporal_comp_ever
+ * @brief Return true if a base value is ever less than a temporal value
+ * @sqlfn ever_lt()
+ * @sqlop @p ?<
+ */
+Datum
+Ever_lt_base_temporal(PG_FUNCTION_ARGS)
+{
+  return EAcomp_base_temporal(fcinfo, &ever_lt_temporal_base);
+}
```

Comment:
Shouldn't we invert the operator too here?
We're inverting the order of the parameters but not the operator, although it is not commutative.

Reply:
The same happens later too